### PR TITLE
fix Issue 14712 - GIT HEAD : std.net.curl regressions

### PR DESCRIPTION
**Pull request by CyberShadow**
https://issues.dlang.org/show_bug.cgi?id=14712

### CODE REVIEW

**Comment by schveiguy** [comment]
Auto-merge toggled on

File: std/net/curl.d [hunk 1]
```diff
@@ -309,8 +309,11 @@ unittest
     if (!netAllowed()) return;
     // No anonymous DigitalMars FTP access as of 2015
     //download("ftp.digitalmars.com/sieve.ds", buildPath(tempDir(), "downloaded-ftp-file"));
-    download("d-lang.appspot.com/testUrl1", buildPath(tempDir(), "downloaded-http-file"));
-    download!(HTTP)("d-lang.appspot.com/testUrl1", buildPath(tempDir(), "downloaded-http-file"));
+    auto fn = buildPath(tempDir(), "downloaded-http-file");
+    download("d-lang.appspot.com/testUrl1", fn);
+    assert(std.file.readText(fn) == "Hello world\n");
```
Comment (ibuclaw) [inline std/net/curl.d]: Shouldn't this be a soft assert as it relies on environmental factors?

Comment (ibuclaw) [inline std/net/curl.d]: Ah, I've just noticed the netAllowed above. :)

**Comment by CyberShadow** [comment]
Crap, was this supposed to be targeting the stable branch?

**Pull request merged**